### DISCUSSION TITLE
OSPOlogyLive Apeldoorn - Roundtable Meeting Notes (Day 2)

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#439] [OSPO Discussions]
Creating a thread for Roundtable leaders to share their meeting notes (leaving a comment) gathered during their roundtable sessions on day 2 (Thursday 29)

**Comment by anajsana** [comment]
## Deep Dive into the Policy Blackbox - Roundtable B (Day 3)

### Section 1: Scope

After the talk of bosch today and the Dutch tax admin got to the question:

Is there a need for a policy to get the culture of sharing working?

Is needed to fill the gap on strategy vs technical? If so, how?

### Section 2: Group intros

* Razvan
* Dutch Tax Admin
* Foundation for Public Code
* Linux Foundation
* Ministry of health, welfare, and sports

### Section 3: Brainstorming 

* Policy is needed for sharing best practices but is important to discuss with developers to also understand their needs 
* Should we just give room to developers to organically grow contributions? should they need guidelines? are policies that critical?
* Need to create a way of continuously working, beyond policies
    * Create a safe environment: provide a safe space to developers organically grow
    * Is about Education
    * How to take responsibility on a corporate level
    * Best practice: having classification levels 
    * Do OSPOs on a governmental level act as facilitators or do we need to do more?
* InnerSource / Open source / Closed Sourced. Does not matter. It's Software Development
*  Under public context: Having a classification level on "all code should be open source" except "XYZ" and having the OSPO facilitate such policy, processes changes
* What does OSPO in public Admin need to be? enables? facilitators? policy-makers?
* Under policy-level context: What happens when nobody in the room says "We need to improve digitalization"?
     *  Go Baby steps: Shifting to a regulation level instead of a legislation level
* Make sure that everyone you are in touch with knows the incubation projects and initiatives going (ambassador programs to translate what the OSPO is doing across departments, ministries, etc)
* How to change the toxic culture of "making mistakes in code is failure" to a healthy culture of "making mistakes in code is a way to improve and become better in your skills". E.g how to avoid "developer champions" becoming demotivated because contributing becomes a threaded environment because of a toxic culture?
* The role of the OSPO is critical on Supporting people who want to contribute AND convincing people who are scared of continuing 

### Section 4: Shared challenges and pain points identified 

* Creating awareness (on strategic level/policy level/technical level): we need metrics to create evidence.
* Facilitate: we need to remove barriers
* Convince managers that open source projects are not a temporary/situational thing, but a structural and continuous baseline
* Make sure we put big efforts into Celebrate Success  and do it through the right channels
* Create a bi-directional connection between policymakers and developers and vice-versa via business cases
* Need for business cases. A way to share this openly (the non-code stuff) the architecture, processes, and best practices, to share with managers.

**Comment by geekygirldawn** [comment]
## Governance Game - Roundtable Day 2

[https://governancegame.publiccode.net/](https://governancegame.publiccode.net/) 

### Playing the Game

* Scenario: 2 municipalities develop an OSS public transport sys. It’s been in use for 1 year. The end users are civil servants. Both orgs are Commissioning.
    * Commissioning = They brought in additional developers on commission to do some of the development because it was too much for the internal developers 
    * 2 orgs -> Development team (in house + vendor) in collaboration with Tech steering -> codebase / back-end / front-end -> Civil Servant
* Calamity: One org wants an additional language
    * This creates tension between the orgs. Adding a language helps the one org, but likely slows down the system for the second org. 
    * Do we need to select new tech to support multiple languages? Do we continue as one project or fork into 2?
        * Just modify what we already have? Makes it more usable in the future.
* Calamity: GDPR compliance - needs to be 100% compliance, who is checking to make sure it’s compliant?
    * Each org will need to verify it - they might have separate policies, and each organization is individually responsible and can’t be delegated to another org.
    * Developers / steering committee will need to make it happen, but each org will need to verify.
* Add a governance body between the 2 organizations and also connected to a new product steering group in addition to technical steering, which moves technical steering closer to dev teams, but still connected to the governing body and in line with product steering.

### Reflecting on the game

* They would like to do flavors / variations on the game. Right now, it’s tailored to public policy orgs, but they would like to expand the cards to be more relevant for other types of orgs. It’s on their roadmap, but they would like help applying it to other sectors.
* Collaborate on the game here: [https://github.com/publiccodenet/governance-game](https://github.com/publiccodenet/governance-game) 
    * They’ve digitized everything, and you can play it online on a digital canvas, but people are less likely to move things around. Takes more than twice the amount of time to play it online.
* The OSS element is hidden, not visible. Where are the communities?
* They often make new calamity cards based on issues that the org they are working with is seeing and wants to focus on.
* The discussion about governance happening before the calamity is helpful in understanding how you might respond.
* Playing this game with everyone involved can be very enlightening because it gets people talking to groups they might not interact with.
* There are video tutorials online for how to run the game.
* When would you do this in the flow of a project? A project usually starts and then there is inevitable friction around who should make a decision, then it’s time to play the game. When you need clarity on the process, hopefully, before the calamity.
* This can help everyone understand how everything / everyone works together. You can start with how it works now and get people working together to improve it.
* Smaller groups (4-5 people) can feel safer and more people get a chance to speak. Fewer people and it’s hard to understand complexity. Too many people and it’s hard to get everyone giving feedback.

**Comment by ghost** [comment]
## Sustainable IT Roundtable Summary

### Introduction

The concept of sustainable IT has gained more attention in the past two years, becoming a significant theme for CTO Office of the Dutch Tax Office.

Attendees discussed diverse perspectives on sustainability IT.

Challenges were identified in defining what constitutes sustainability, especially in terms of its impact on the climate.

Key points included:
- The digital footprint of IT systems and the imperative to use resources sustainably.
- The role of optimizing code, power usage, and collaboration in achieving sustainability.
- The significant contribution of open-source software in sharing algorithms and knowledge to enhance resource efficiency.

### Challenges Identified

Issues in adopting sustainable tools and practices include:
- Inconsistent use of energy resources.
- Lack of insight and prioritization for sustainable standardization.
- Conservative mindsets and resistance to change.
- The high energy consumption associated with AI and the prevalence of a throwaway culture in technology.

### Solutions and Moving Forward

**Key suggestions to improve sustainability in IT included:**
- Recognizing and rewarding governmental efforts in sustainability.
- Standardizing the use of sustainable hardware across government entities.
- Emphasizing the importance of sustainability and integrating it into policy and accountability measures.

### Discussion Outcomes

- The roundtable emphasized the need for simple yet effective legislation.
- The concept of demand shaping was discussed as a means to optimize energy usage based on availability.
- Strategies for making sustainability visible and incentivized within organizations were debated, including the use of visual feedback and efficient configuration of virtual machines.

### Top 3 Solutions Voted

**Problem - Lack of facts, figures, measurements**
**Solution**:
- Emphasize continuous measurement of power usage and allocation of workloads to minimize idle machine time.
- Enhance reporting and feedback mechanisms to inform policy decisions, with a focus on energy consumption.

**Problem: not a policy objective:**
- Integrate sustainability into annual reports and accountability frameworks to recognize its importance.
- Advocate for sustainability as a key agenda for government ministers and agencies.

**Problem: fear of change:**
- Encourage agencies to set sustainability benchmarks, facilitating the adoption of sustainable practices by lowering barriers.
- Focus on reducing the demand for excessive computing power and question the necessity of current usage levels.
- Dashboarding
- Create awareness on all levels.

![IMG_4628](https://github.com/todogroup/ospology/assets/85950159/e68179fa-7375-4926-b4a7-a16bd29ee142)
![IMG_4629](https://github.com/todogroup/ospology/assets/85950159/5dab2f2b-7849-4ebc-b993-fe9db6b2f622)
